### ISSUE TITLE
Preload the next chapter during reading

### ISSUE THREAD

**Issue by philliphs** [tachiyomiorg/tachiyomi#7266]
### Describe your suggested feature

For chapter that is not pre-downloaded, pages are loaded until the end of chapter is currently reading.

There should be at least an option on settings to load the next chapter straight away when the current chapter finishes loading.

For example opening chapter 1 page 1 initiates background loading of chapter 1 from page 1 to page 10 (assuming end of chapter is page 10).
Current behaviour is background loading finishes at chapter 1 page 10.
I suggest background loading keeps working to load until chapter 2 page 10 (end of chapter 2), therefore there will be no visible loading at the end of chapter 1 moving to chapter 2. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by philliphs** [comment]
I wonder if there can be an option to preload the next chapter earlier.
I'm still experiencing wait on loading the next chapter half of the time, maybe because of the distance of the sources server from my country
